style.css

```diff
@@ -0,0 +1,128 @@
+body {
+  background-image: url('images/background.jpg');
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center center;
+}
+
+.col {
```

Comment:
Let's not re-style the bootstrap classes, rather add a new class that adopts this styles.